packages/wallet/package.json

```diff
@@ -16,12 +16,10 @@
     "build": "run-s clearContracts build:typescript build:webpack",
     "clearContracts": "rm -rf ./build/contracts",
     "contracts:compile": "node ./bin/compile.js",
-    "lint:check": "yarn run tslint --config tslint.json  './**/*.{ts,js,tsx,jsx}'",
-    "lint:write": "yarn run tslint --config tslint.json  './**/*.{ts,js,tsx,jsx}' --fix ",
+    "lint:check": "npx eslint \"*/**/*.ts\" --cache",
```

Comment:
I would be careful with this `--cache` option. When testing this branch I noticed it caused issues when deleting built contract artifacts, running the linter, and then rebuilding them, and running it again. The linter would fail despite the contract artifacts existing.

Reply:
I agree. If that happens I just run `lint:write` :)
Cache can save a lot of time when linting constantly. 
This is open for discussion.